services/src/main/java/org/keycloak/protocol/oid4vc/issuance/signing/VCSigningServiceProviderFactory.java

```diff
@@ -44,13 +44,15 @@ public interface VCSigningServiceProviderFactory extends ComponentFactory<Verifi
 
     public static ProviderConfigurationBuilder configurationBuilder() {
         return ProviderConfigurationBuilder.create()
-                .property(SigningProperties.KEY_ID.asConfigProperty());
```

Comment:
If removed here, it should be added to the individual providers for Jwt, SdJwt or LD. Why not leaving it here, but just removing the required-constraint in l53?

Reply:
Resolved with [39100ed](https://github.com/keycloak/keycloak/commit/39100edc1281422399b045012acb3cb27062fbf6)